src/utils/jwt.js

```diff
@@ -0,0 +1,97 @@
+/* eslint-env serviceworker */
```

Comment:
Copied from other projects. Will create ticket to get these into a lib!

Reply:
I find it a weird setup to validate user:password if we don't have users. JWT kind of makes sense and is super small setup

Reply:
We'll probably have multiple services that use this though and they'll have different JWT's - they are the users. It just seems overkill to use JWTs.

Can you at least provide some instruction in the README of how to create a JWT so you can use the service in dev? Maybe a script to generate one for each environment?

Reply:
My goal was to keep auth simple until we need something fancy, or decide on a general strategy for auth between services. That meant in this context, we would have single JWT to be used. If/when we want to have multiple services acting as users and interacting with other services, we will need to have an auth service shared by all the services with any auth system we put in place.

If we go via HTTP basic auth, we can just use wrangler secrets to store user:pw, but we will need to change this for more users over time... 

For dev, I would just use Miniflare to spin up the worker with needed global variables. But yes, a script to generate the JWT seems a good call.

Reply:
I was going to suggest UCANs, but I do realize that supporting all of the UCAN spec might seem like an overkill. I'll try to write a proposal of how we can make do simplified UCAN flows without complicated key exchange schemes.

Reply:
things would never go around that worker thats the point, we have a worker similar to the aws api gateway to route requests and call the ucan worker if the request needs auth